docs/content/preview/yugabyte-voyager/migrate/migrate-steps.md

```diff
@@ -18,7 +18,7 @@ The following page describes the steps to perform and verify a successful offlin
 
 ## Migration workflow
 
-![Offline migration workflow](/images/migrate/migration-workflow.png)
+![Offline migration workflow](/images/migrate/migration-workflow-new.png)
```

Comment:
The second step is not correct, it should be-
EXPORT:  Export Schema-> Analyze/Modify -> Export Data